README.md

```diff
@@ -132,7 +132,32 @@ TodoItem.acts_as_list_no_update do
 end
 ```
 In an `acts_as_list_no_update` block, all callbacks are disabled, and positions are not updated. New records will be created with
- the default value from the database. It is your responsibility to correctly manage `positions` values. 
+ the default value from the database. It is your responsibility to correctly manage `positions` values.
+
+Also you can pass an argument as array of extracted calsses to disable from database updates. It might be any ActiveRecord class that is able to acts as list.
```

Comment:
Should read:

You can also pass an array of classes as an argument to disable database updates on just those classes. It can be any ActiveRecord class that has `acts_as_list` enabled.